doc/types/verify_info.qbk

```diff
@@ -0,0 +1,226 @@
+  ]
+  [
+    [[link nudb.ref.nudb__verify_info.hist `v.hist`]]
+    [std::array< std::size_t, 10 >]
+    [
+      A histogram of the number of buckets having N spill records. 
+    ]
+  ]
+  [
+    [[link nudb.ref.nudb__verify_info.key_count `v.key_count`]]
+    [std::uint64_t]
+    [
+      The number of keys found. 
+    ]
+  ]
+  [
+    [[link nudb.ref.nudb__verify_info.key_file_size `v.key_file_size`]]
+    [std::size_t]
+    [
+      The size of the key file. 
+    ]
+  ]
+  [
+    [[link nudb.ref.nudb__verify_info.key_path `v.key_path`]]
+    [path_type]
+    [
+      The path to the key file. 
+    ]
+  ]
+  [
+    [[link nudb.ref.nudb__verify_info.key_size `v.key_size`]]
+    [std::size_t]
+    [
+      The size of each key, in bytes. 
+    ]
+  ]
+  [
+    [[link nudb.ref.nudb__verify_info.load_factor `v.load_factor`]]
+    [float]
+    [
+      The target load factor used in the key file. 
+    ]
+  ]
+  [
+    [[link nudb.ref.nudb__verify_info.overhead `v.overhead`]]
+    [float]
+    [
+      The data amplification ratio. 
+    ]
+  ]
+  [
+    [[link nudb.ref.nudb__verify_info.pepper `v.pepper`]]
+    [std::uint64_t]
+    [
+      The salt fingerprint. 
+    ]
+  ]
+  [
+    [[link nudb.ref.nudb__verify_info.salt `v.salt`]]
+    [std::uint64_t]
+    [
+      The salt used in the key file. 
+    ]
+  ]
+  [
+    [[link nudb.ref.nudb__verify_info.spill_bytes `v.spill_bytes`]]
+    [std::uint64_t]
+    [
+      The number of bytes occupied by spill records in use. 
+    ]
+  ]
+  [
+    [[link nudb.ref.nudb__verify_info.spill_bytes_tot `v.spill_bytes_tot`]]
+    [std::uint64_t]
+    [
+      The number of bytes occupied by all spill records. 
+    ]
+  ]
+  [
+    [[link nudb.ref.nudb__verify_info.spill_count `v.spill_count`]]
+    [std::uint64_t]
+    [
+      The number of spill records in use. 
+    ]
+  ]
+  [
+    [[link nudb.ref.nudb__verify_info.spill_count_tot `v.spill_count_tot`]]
+    [std::uint64_t]
+    [
+      The total number of spill records. 
+    ]
+  ]
+  [
+    [[link nudb.ref.nudb__verify_info.uid `v.uid`]]
+    [std::uint64_t]
+    [
+      The unique identifier. 
+    ]
+  ]
+  [
+    [[link nudb.ref.nudb__verify_info.value_bytes `v.value_bytes`]]
+    [std::uint64_t]
+    [
+      The total number of bytes occupied by values. 
+    ]
+  ]
+  [
+    [[link nudb.ref.nudb__verify_info.value_count `v.value_count`]]
+    [std::uint64_t]
+    [
+      The number of values found. 
+    ]
+  ]
+  [
+    [[link nudb.ref.nudb__verify_info.version `v.version`]]
+    [std::size_t]
+    [
+      The API version used to create the database. 
+    ]
+  ]
+  [
+    [[link nudb.ref.nudb__verify_info.waste `v.waste`]]
+    [std::size_t]
+    [
+      The fraction of the data file that is wasted. 
+    ]
+  ]
+  [
+    [[link nudb.ref.nudb__verify_info.verify_info `verify_info()`]]
+    [std::size_t]
+    [
+      Default constructor.
+    ]
+  ]
+]
+[heading Header]
+[@../../include/nudb/verify.hpp `#include <nudb/verify.hpp>`]
```

Comment:
Right - but it doesn't generate a link to the actual header file.

The version I made is much shorter for not having the private variables.  So it's simple for the user to understand.  But for those who are gluttons for punishment one can open the header file itself to see all the implementation details